newrelic.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ["Facelift"],
```

Comment:
I think this should be based on the `APP_ENV` environment variable so the New Relic dashboard can track both QA and prod.

Reply:
Ah, yes of course, that's definitely required to distinguish the two.